packages/obonode/obojobo-chunks-excerpt/changes/normalize-node.js

```diff
@@ -0,0 +1,73 @@
+import { Editor, Node, Element, Transforms, Text } from 'slate'
```

Comment:
With any luck #2068 will make this a non-issue since users won't be able to insert excerpts into other excerpts, but in the meantime I've adjusted all of the important CSS rules to be extremely specific to avoid as much style bleeding between nested components as possible.

My opinion is that it shouldn't be easy to do, but if somebody **really** wants an excerpt inside another excerpt they should still be able to go into the XML or JSON editors and do it manually.